rootfs/ReadMe.md

```diff
@@ -0,0 +1,11 @@
+# FrOST Root FS
+
+## Key Files
+ - `build_rootfs.sh`
+ - `setup_rootfs.sh`
+ - `packages`
+
+## Building the Root FS with Docker
+1. Install [Docker for Windows](https://docs.docker.com/docker-for-windows/install/)
+2. Open WSL and Navigate to `[FrOST Repos]\utils\rootfs\`
+3. Run `./build_rootfs.sh`
```

Comment:
I (and the jenkinfiles) run `docker_build.sh`

Reply:
Updated with 2b9b109